src/messaging/ExchangeDelegate.h

```diff
@@ -100,5 +100,31 @@ class DLL_EXPORT ExchangeDelegate
     virtual ExchangeMessageDispatch & GetMessageDispatch() { return ApplicationExchangeDispatch::Instance(); }
 };
 
+/**
+ * @brief
+ *   This class handles unsolicited messages. The implementation can peek the message and creating a new exchange for it or not.
+ */
+class DLL_EXPORT ExchangeAcceptor
```

Comment:
This should be called UnsolicitedMessageHandler.

Reply:
Done